### ISSUE TITLE
It is so difficult to re-apply kubeflow. The scripts should skip deleting namespace. 

### ISSUE THREAD

**Issue by b675987273** [kubeflow/kubeflow#4613]
/kind bug

**What steps did you take and what happened:**
[When i failed apply kubeflow and try to delete it. The namespaces can t terminate naturally]
```shell
istio-system                           Terminating   3h19m
knative-serving                        Terminating   3h19m
kubeflow                               Terminating   27d
```

**What did you expect to happen:**
I expect that kubflow cam be re-apply more easily. 

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- Kubeflow version: (version number can be found at the bottom left corner of the Kubeflow dashboard):v0.7
- kfctl version: (use `kfctl version`):v 0.7.0
- Kubernetes platform: 
- Kubernetes version: (use `kubectl version`):1.14
- OS (e.g. from `/etc/os-release`):

**Comment by yanniszark** [comment]
@b675987273 thanks for reporting!
Personally, I don't think we should skip deleting the namespace.
This seems like a symptom of failing to cleanup something properly and we should investigate the root cause.

**Comment by yantriks-edi-bice** [comment]
I deployed abc and was facing many issues so had to delete and re-apply facing yet more issues which were insurmountable. So I gave up and created xyz which worked well. I did not yet delete abc in order to compare between the two and learn about some of the issues I faced with abc. Then I proceeded to delete abc which indeed deleted the kubeflow namespace. So many resources cannot be deleted if they're in use but for some reason this one goes through. So now xyz is broken as well. I agree namespace deletion should not be skipped but specifying namespace should be part of the set up.
